### DISCUSSION TITLE
On the mapping of the N-DMBI molecule

### DISCUSSION THREAD

**Discussion by Clare-xiong** [ricalessandri/Martini3-small-molecules#31] [Q&A] [answered]
Hi. I try to map the N-DMBI, but I always miss one carbon in the benzene like this
![N-DMBI](https://user-images.githubusercontent.com/85223463/231767469-ff233c5e-efe0-4c00-a98c-8d9410ab83b6.jpg)
Could you help me mapping N-DMBI correctly?

**Comment by ricalessandri** [comment, marked as answer]
@Clare-xiong - I'm sorry, this post slipped through my notifications.

Your mapping is almost correct. That missing carbon in the benzene should be mapped together with the -N(CH3)2 (the S-bead you highlighted in green). At first it may look as a slightly "overmapped" S-bead (4 non-hydrogen atoms); however, because it is fully branched, this is the recommended way to map it - see:

_"fully branched fragments should usually use beads of smaller size (the rational being that the central atom of a branched group is buried, that is, it is not exposed to the environment, reducing its influence on the interactions); for example, a neopentane group contains 5 non-hydrogen atoms but, as it is fully branched, you can safely model it as a regular bead."_

from [section 2 of this tutorial](http://cgmartini.nl/index.php/2021-martini-online-workshop/tutorials/560-5-parametrization-of-a-new-small-molecule).
